### DISCUSSION TITLE
pytest fixture import issue

### DISCUSSION THREAD

**Discussion by johnsturgeon** [astral-sh/ruff#15088] [Q&A] [answered]
I'm being told by `ruff` that an imported test fixture is not being used.. but.. well... it is.

<img width="738" alt="image" src="https://github.com/user-attachments/assets/8e6dd78f-99be-4f97-a2bb-9063f5336e26" />

Any thoughts?

**Comment by calumy** [comment, marked as answer]
See this stack overflow answer to a similar question: https://stackoverflow.com/questions/75647682/how-can-i-resolve-flake8-unused-import-error-for-pytest-fixture-imported-from

**Reply by johnsturgeon** [reply]
Fantastic, thank you!

**Comment by johnsturgeon** [comment]
Answered here: https://github.com/astral-sh/ruff/discussions/15088#discussioncomment-11633678
